afterburners/fxmanifest.lua

```diff
@@ -0,0 +1,4 @@
+fx_version 'bodacious
```

Comment:
syntax error, bodacious is also outdated.

Reply:
Ah yeah i missed that my bad and its better then __resource.lua